lib/extensions/string_extension.rb

```diff
@@ -29,6 +29,12 @@ def strftime(format)
     Time.zone.parse(self).strftime(format)
   end
 
+  def numeric?
```

Comment:
💭 Consider renaming this method to something that describes more accurately its behaviour (e.g. `has_numeric_ending?`, `ends_with_numeric?`), as the method name, as is, seems misleading.

Reply:
I'm removing it anyway as it's no longer needed but, in truth, it did check the entire string.  I split it in the, also deleted, concern
```ruby
if role.split('_').last.numeric?
```
It was only checking the numeracy of the final string in the array... but anyway, it's going! :)